keps/sig-node/2238-liveness-probe-grace-period/README.md

```diff
@@ -198,11 +189,13 @@ quickly.
   available behind a feature flag.
 - Appropriate tests are written.
 
-_Below graduation criteria are tentative._
-
 #### Beta
 
 - Feature flag is defaulted on.
+- Remove feature gate from [kubelet](https://github.com/kubernetes/kubernetes/pull/99375#issuecomment-794680869).
```

Comment:
To support n-2 skew, I think we need do the following:

- the feature gate is not present on the kubelet
- the feature gate is not defaulted to on until kube-apiserver is +2 versions of the kubelet having the support.

Reply:
My concern is about the scenario where:
 - feature gate is not present in kubelet
 - user turns on feature in apiserver, sets the new field
 - user turns off feature in apiserver

As written, in this scenario, the feature would still be active for the workloads with the value set, because the kubelet cannot turn off that feature, and it still sees the new field.

Reply:
I still don't see this scenario handled properly if you remove the feature gate from kubelet early.